.github/workflows/ci.yml

```diff
@@ -81,7 +83,7 @@ jobs:
           chmod a+x /opt/devkitpro/tools/bin/makerom
 
       - name: Install Linux dependencies (apt-get)
-        if: ${{ matrix.linux }}
+        if: ${{ matrix.linux || matrix.android }}
```

Comment:
Is this still needed? You're no longer doing cmake -> gradle -> cmake.

Reply:
What error?
We're installing x11, opengl and qt6 for Linux here, things Android does not need.

Reply:
I understand why it fails because of  your initial run with `-DSDL_ANDROID=ON`.
That should be avoided.